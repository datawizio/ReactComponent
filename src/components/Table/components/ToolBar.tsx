import * as React from "react";

import "../index.less";

const ToolBar: React.FC = props => {
  return <div className="table-toolbar">{props.children}</div>;
};

export default ToolBar;
