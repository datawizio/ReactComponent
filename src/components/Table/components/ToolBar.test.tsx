import React from "react";
import { shallow } from "enzyme";
import ToolBar from "./ToolBar";

const setUp = (props?) => shallow(<ToolBar {...props} />);

describe("ToolBar component", () => {
  let component;
  let instance;

  beforeEach(() => {
    component = setUp();
    instance = component.instance();
  });

  it("rendered ToolBar corectly", () => {
    expect(component).toMatchSnapshot();
  });

  it("ToolBar has 1 toolbaR class ", () => {
    const cellClass = component.find(".dw-table__toolbar");
    expect(cellClass.length).toBe(1);
  });
});
