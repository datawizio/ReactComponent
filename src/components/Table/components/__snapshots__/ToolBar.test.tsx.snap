// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolBar component rendered ToolBar corectly 1`] = `
<div
  className="dw-table__toolbar"
/>
`;
